src/mumble/WASAPI.cpp

```diff
@@ -1065,7 +1065,9 @@ void WASAPIOutput::run() {
 	if (pwfx)
 		CoTaskMemFree(pwfx);
 
-	setVolumes(pDevice, false);
+	if (pDevice) {
+		setVolumes(pDevice, false);
```

Comment:
setVolumes is called in other contexts where you don't do the null check.
Maybe it should be moved into the function itself?